airbyte-integrations/connectors/source-looker/unit_tests/test_URL.py

```diff
@@ -0,0 +1,44 @@
+import re

```

Comment:
Naming shall conform to [PEP8 Naming Conventions](https://www.python.org/dev/peps/pep-0008/#prescriptive-naming-conventions).

Reply:
changed filename and tests as per pep8 standards